### PR TITLE
chore(deps): version update

### PR DESCRIPTION
**Pull request by Tyagi-Sunny**
## Description
loopback version update

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,29 +49,29 @@
...
     "@semantic-release/commit-analyzer": "^9.0.2",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/npm": "^9.0.1",
     "@semantic-release/release-notes-generator": "^10.0.3",
     "@types/dotenv": "^8.2.0",
     "@types/moment-timezone": "^0.5.30",
-    "@types/node": "^18.11.9",
-    "@typescript-eslint/eslint-plugin": "^7.8.0",
-    "@typescript-eslint/parser": "^7.8.0",
+    "@types/node": "^16.18.119",
```
Comment (yeshamavani) [inline package.json:72]: why is the version downgraded

**Review by yeshamavani — CHANGES_REQUESTED** [review]
image

Also include changes of remaining Prs in this

File: package.json (hunk 1 above)
Comment (Tyagi-Sunny) [reply package.json]: done

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**